doc/concepts/promotion.rst

```diff
@@ -1,6 +1,19 @@
+=====================
```

Comment:
```suggestion
```
Does not seem to be the case on the other files. Rst is supposed to take the top level title as the page title.